ARMTemplates/AzFunctionApp-Consumption.parameters.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appName": {
+            "value": "[concat('fnapp', uniqueString(resourceGroup().id))]"
```

Comment:
Perhaps we can make these default values more descriptive to the actual project? Like just a prefix that's more descriptive than `fnapp`?

Reply:
there is a storage acc char limit when using BroomBot as the prefix. But ill look into that.